auth_otp/management/commands/cleanup_otp.py

```
@@ -0,0 +1,102 @@
+
+class Command(BaseCommand):
+    """
+    پاکسازی OTP ها و توکن‌های منقضی شده
+    
+    استفاده:
+        python manage.py cleanup_otp
+        python manage.py cleanup_otp --dry-run
+    """
+    help = 'پاکسازی OTP ها و توکن‌های منقضی شده'
+    
+    def add_arguments(self, parser):
```

Ruff (0.12.2)

Missing return type annotation for public function `add_arguments`

Add return type annotation: `None`

(ANN201)